### ISSUE TITLE
Add an API endpoint test

### ISSUE THREAD

**Issue by Azoorb** [stancl/tenancy-docs#264]
I find that there is a lack of a complete test example in the documentation, especially for testing an API endpoint.

Indeed, I am using multi-database tenancy with domain routing, and testing something like `$this->get('/auth/login')` is not straightforward because we cannot do `$this->get('fakedomain.com/auth/login')`, it result in a 404 response.

Perhaps adding an example in the documentation could make testing clearer.
Thank you for this great package!

**Comment by cgarciagarcia** [comment]
I agree. It's not enought clear, how to do test in generalusing the package..
